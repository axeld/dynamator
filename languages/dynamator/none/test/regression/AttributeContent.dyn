<dynamator language="none">
  <class name="right">
    <attr name="align">
      <content>right</content>
    </attr>
  </class>
  <class name="left">
    <attr name="align">
      <discard/>
    </attr>
  </class>
</dynamator>
