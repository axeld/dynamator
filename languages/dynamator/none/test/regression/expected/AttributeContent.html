<html>
  <head>
    <title>Test Attribute Content</title>
  </head>
  <body>
    <p class="right" align="right">This should be right-justified.</p>
    <p class="right" align="right">This should be right-justified.</p>
    <p class="left">This should be left-justified.</p>
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
